src/torch_onnx/_patch.py

```diff
@@ -48,41 +48,48 @@ def _from_dynamic_axes_to_dynamic_shapes(
     """
     # https://github.com/pytorch/pytorch/pull/128371
     if dynamic_axes is None:
-        return None
+        return None, input_names
```

Comment:
Is returning input_names expected? Looks like we expect that to be `dict[str, Any] | None`

Reply:
Done